contracts/src/ProtocolAdapter.sol

```diff
@@ -49,9 +49,8 @@ contract ProtocolAdapter is IProtocolAdapter, ReentrancyGuardTransient, Commitme
     error RiscZeroVerifierStopped();
 
     error CalldataCarrierKindMismatch(bytes32 expected, bytes32 actual);
-    error CalldataCarrierAppDataMismatch(bytes32 expected, bytes32 actual);
-    error CalldataCarrierLabelMismatch(bytes32 expected, bytes32 actual);
-    error CalldataCarrierCommitmentNotFound(bytes32 commitment);
+    error CalldataCarrierCmMismatch(Resource resource, bytes32 expected, bytes32 actual);
```

Comment:
Is there a reason not to write the full name (`CalldataCarrierCommitmentMismatch`)? This shorthand is unusual.